src/components/organisms/demo-tile/demo.js

```diff
@@ -46,13 +55,15 @@ export default {
                             <Text>Large content</Text>
                         </View>
                     )}
+                    navigation={{}}
+                />
+            ),
+            renderHeader: (props: HeaderProps) => (
```

Comment:
Is there also a demo for demonstrating how to use `renderHeader` to actually render something like a header? Because this demo only show how to use `renderHeader` to render an absolute positioned floating action button, which feels like kind of abusing the `renderHeader` method?